### DISCUSSION TITLE
Logs: Make time format configurable

### DISCUSSION THREAD

**Discussion by svennergr** [grafana/grafana#64389] [Dashboards, Panels & Visualizations]
In https://github.com/grafana/grafana/pull/64372 we implemented showing millisecond time precision in log lines, however it might be useful to have those time columns configurable.

E.g. configure a `timeformat` in the panel/visualisation setting and use that to show times in the LogRow.

The default time format can already be changed with those configuration items: https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#date_formats
![image](https://user-images.githubusercontent.com/8092184/223713735-76de910c-63e3-4b30-808f-e8457b0225f5.png)

However, this will change the time format not only for logs visualisations, and not all users have access to Grafana's configuration.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82794. Feel free to continue the discussion around this there. Thank you!
